content/en/mobile/_index.md

```diff
@@ -83,11 +83,120 @@ On the Incidents page, you can view, search and filter all incidents that you ha
+- Search for "Datadog" widgets.
+- Select your prefered size (small, medium, or large).
+- Drag the widget to your desired, on-screen location.
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Long press on your home screen.
+- Tap the "Widgets" button on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
+- Drag the widget to your desired, on-screen home screen.
+- Resize the widget to fit your preference.
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Edit an Open Incidents widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+- Long press on the widget to configure.
+- Tap "Edit Widget."
+- Tap "Choose" next to the Organzation label to fetch open incidents from the selected organization.
+- Tap "SEV-1 and SEV-2" next to the Severities label to specify severity filters.
+- Tap "Both" next to the Customer Impacted label to filter on open incidents that have impacted customers.
+- Type in the "Type additional filters" text box to specify any further filtering.
+- Tap "Ordering" to specify the order of how incidents will be listed.
+- Tap out of the widget to validate your selection and exit the configuration screen.
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Tap the widget title to configure.
+- Tap "Organization" to fetch open incidents from the selected organization.
+- Tap "Severities" to specify severity filters.
+- Tap "Customer impacted" to filter on open incidents that have impacted customers.
+- Tap "Query" to specify any further filtering.
+- Tap "Sorted by" to specify the order of how incidents will be listed.
+- Tap "Save" or "Apply" to validate your selection and exit the configuration screen.
+- Long press and resize the widget to fit your preference.
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Display Open Incidents from multiple organizations
+
+You can display open incidents from multiple organizations on your mobile home screen. 
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+- Tap "Choose" next to the Organzation label to fetch open incidents from the selected organization.
+
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Tap the widget title to configure.
+- From the configuration screen, tap "Organization".
+- Select a new organization (You might need to sign in).
+- Size the widget to fit your preference.
+- Tap "Save" or "Apply". 
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Delete an Open Incidents widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+Delete a widget by tapping the "-" button at the top left of the widget when editing your home screen, or by long pressing on the widget and selecting "Remove Widget".
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+Delete a widget by long pressing, dragging, and dropping the widget to the "Remove" button.
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
 ### SLOs widget
 
 {{< img src="mobile/slo_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
```

Comment:
Can we please update the alt text on this to be more accessible and descriptive while we're updating this page?
```suggestion
{{< img src="mobile/slo_widget.png" alt="Application Uptime SLO widgets displayed on Android and iOS devices" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
```